### DISCUSSION TITLE
Consistent hook naming?

### DISCUSSION THREAD

**Discussion by erganemic** [alphapapa/hammy.el#24] [Q&A]
Don't think this warrants an issue, but I was curious why all hooks were named with a `-hook` suffix, with the exception of `hammy-stopped`; is this a break from convention that implies something semantically different about that particular hook?

**Comment by alphapapa** [comment]
I don't know.  You can see in def1c65571ae9b3ed27db0ff9dd9868a8f000ba9 where I made that change two years ago, to match the name of a slot in the struct.  It probably should end in `-hook` since it's an option.  Feel free to make an issue suggesting it.
